README.md

```diff
@@ -1,135 +1,170 @@
-# Diarizers
+# 🤗 Diarizers
 
-Diarizers is a library for fine-tuning [`pyannote`](https://github.com/pyannote/pyannote-audio/tree/develop) speaker diarisation models using Hugging Face tools. It provides scripts and tools to: 
-
-- Convert Speaker Diarization datasets into Hugging Face datasets compatible with the `diarizers` library. 
-- Fine-tune speaker diarization models using the Hugging Face `Trainer`. 
-- Test the models at inference time. 
-- Convert the fine-tuned models into a format compatible with `pyannote`.  
-
-> [!IMPORTANT]
-> For now, this library can only be used to fine-tune the [segmentation model](https://huggingface.co/pyannote/segmentation-3.0) from the [speaker diarization pipeline](https://huggingface.co/pyannote/speaker-diarization-3.1). 
-> Future work will aim to help optimise the hyperparameters of the entire pipeline. 
+🤗 Diarizers is a library for fine-tuning [`pyannote`](https://github.com/pyannote/pyannote-audio/tree/main) speaker 
```

Comment:
Intro just needs to be one paragraph that summarises the efforts. As punchy as possible. Feel free to iterate on the suggestion given here.